project/AppDependencies.scala

```diff
@@ -33,7 +31,7 @@ private object AppDependencies {
     def apply() =
       new TestDependencies {
         override lazy val test = Seq(
-          "org.scalatest"          %% "scalatest"          % "3.0.9",
+          "org.scalatest"          %% "scalatest"          % "3.0.9" % scope,
```

Comment:
What does the % Scope add?

Reply:
not sure, the bot said to do it so I done it

Reply:
hahha yeah makes sense